gordo_components/model/factories/model_factories_utils.py

```diff
@@ -13,3 +18,31 @@ def hourglass_calc_dims(compression_factor, encoding_layers, n_features):
         round(n_features - (i * average_slope)) for i in range(1, encoding_layers + 1)
     ]
     return dims
+
+
+def align_kwargs(ignore: List[str] = []):
```

Comment:
I know that its hard to delete written code, but this is no longer needed right? Maybe keep it in your backpocket for the next time it might be needed?